set.mm

```diff
@@ -99162,6 +99162,137 @@ a Cantor normal form (and injectivity, together with coherence
+      VNXRGBVOTVPVQVTVRVSBWAWBWPXCXEXLUTZWGWHWQUUFWOXBGBBDEWCWDWEWF $.
+  $}
+
+  ${
+    frrlem15.1 $e |- B = { f | E. x ( f Fn x /\
+                             ( x C_ A /\ A. y e. x Pred ( R , A , y ) C_ x ) /\
+                             A. y e. x ( f ` y ) =
+                                     ( y G ( f |` Pred ( R , A , y ) ) ) ) } $.
+    frrlem15.2 $e |- F = frecs ( R , A , G ) $.
+
+    $d A a f x y g h u v $.  $d R a f x y g h u v $.  $d G a f x y g h u v $.
+    $d B a $.
+    $( Lemma for general well-founded recursion.  Two acceptable functions are
+       compatible.  (Contributed by Scott Fenton, 11-Sep-2023.) $)
+    frrlem15 $p |- ( ( ( R Fr A /\ R Se A ) /\
+      ( g e. B /\ h e. B ) ) -> ( ( x g u /\ x h v ) -> u = v ) ) $=
+      ( va cv wbr wa cres wceq cdm cin wfr wse wcel weq vex breldm adantr elind
+      adantl id brresi anbi12i an4 bitri syl21anbrc wfn cfv cpred co wral inss1
+      wss frrlem3 sstrid ad2antrl simpll frss sylc simplr sess2 frrlem4 reseq2i
+      wb incom fneq12 mp2an fveq1i predeq2 ax-mp reseq12i oveq2i eqeq12i sylibr
+      raleqbii ancoms frr3g syl211anc breqd biimprd wfun wi frrlem2 funresd wal
+      wrel dffun2 2sp sps simplbiim syl sylan2d syl5 ) APZDPZIPZQZXECPZJPZQZRZX
+      EXFXGXGUAZXJUAZUBZSZQZXEXIXJXOSZQZRZEGUCZEGUDZRZXGFUEZXJFUEZRZRZDCUFZXLXE
+      XOUEZYIXLXTXLXMXNXEXHXEXMUEXKXEXFXGAUGZDUGZUHUIXKXEXNUEXHXEXIXJYJCUGZUHUK
+      UJZYMXLULXTYIXHRZYIXKRZRYIYIRXLRXQYNXSYOXOXEXFXGYKUMXOXEXIXJYLUMUNYIXHYIX
+      KUOUPUQYGXSXEXIXPQZXQYHYGYPXSYGXPXRXEXIYGXOGUCZXOGUDZXPXOUROPZXPUSYSXPXOG
+      YSUTZSLVATOXOVBRZXRXOURZYSXRUSZYSXRYTSZLVAZTZOXOVBZRZXPXRTYGXOEVDZYAYQYDU
+      UIYCYEYDXOXMEXMXNVCABEFGHILMVEVFVGZYAYBYFVHXOEGVIVJYGUUIYBYRUUJYAYBYFVKXO
+      EGVLVJYFUUAYCABEFGHIJLOMVMUKYFUUHYCYEYDUUHYEYDRXJXNXMUBZSZUUKURZYSUULUSZY
+      SUULUUKGYSUTZSZLVAZTZOUUKVBZRUUHABEFGHJILOMVMUUBUUMUUGUUSXRUULTXOUUKTZUUB
+      UUMVOXOUUKXJXMXNVPZVNZUVAXOUUKXRUULVQVRUUFUUROXOUUKUVAUUCUUNUUEUUQYSXRUUL
+      UVBVSUUDUUPYSLXRUULYTUUOUVBUUTYTUUOTUVAXOUUKGYSVTWAWBWCWDWFUNWEWGUKOXOGXP
+      XRLWHWIWJWKYGXPWLZXQYPRYHWMZYDUVCYCYEYDXOXGABEFGHILMWNWOVGUVCXPWQUVDCWPDW
+      PZAWPUVDADCXPWRUVEUVDAUVDDCWSWTXAXBXCXD $.
+  $}
+
+  ${
+    $d R w z $.  $d A w z $.
+    $( Lemma for general well-founded recursion.  Establish a subset
+       relationship.  (Contributed by Scott Fenton, 11-Sep-2023.) $)
+    frrlem16 $p |- ( ( ( R Fr A /\ R Se A ) /\ z e. A ) ->
+       A. w e. TrPred ( R , A , z )
+       Pred ( R , A , w ) C_ TrPred ( R , A , z ) ) $=
+      ( wfr wse wa cv wcel cpred ctrpred wss simplr simpllr simpr trpredtr sylc
+      jca ralrimiva ) CDEZCDFZGZAHZCIZGZCDBHZJCDUCKZLZBUGUEUFUGIZGZUDUAGUIUHUJU
+      DUAUBUDUIMTUAUDUINRUEUIOCDUCUFPQS $.
+  $}
+
+  ${
+    $d F x y z u v a b c f g h $.  $d R x y z u v a b c f g h $.
+    $d A x y z u v a b c f g h $.  $d G x y z u v a b c f g h $.
+    frr.1 $e |- F = frecs ( R , A , G ) $.
+    $( Law of general well-founded recursion, part one.  This may look like a
```

Comment:
"This may look like" because "It is", correct ?

Can we rather write something like: "This and the following two theorems frr2 and frr3 drop the partial order requirement from pofr1 [or whatever its labels], pofr2 and pofr3, which requires using the axiom of infinity." ?

Reply:
Done